source/virtualBuffers/gecko_ia2.py

```diff
@@ -6,22 +6,26 @@
 from typing import (
 	Iterable,
 	Optional,
+	Tuple,

```

Comment:
Importing Tuple from typing is deprecated in favor of just subscripting the builtin tuple class.